COMETwebapp/SessionManagement/SessionAnchor.cs

```diff
@@ -188,11 +193,101 @@ public async Task RefreshSession()
         public void SwitchDomain(DomainOfExpertise? DomainOfExpertise)
         {
             var iteration = this.GetIteration();
-            if(iteration != null)
+            if (iteration != null)
             {
                 this.CurrentDomainOfExpertise = DomainOfExpertise;
                 this.Session.SwitchDomain(iteration.Iid, DomainOfExpertise);
             }
         }
+
+        /// <summary>
+        /// Write new Things in the session
+        /// </summary>
+        /// <param name="thingsToCreate">List of Things to create in the session</param>
+        public async Task CreateThings(IEnumerable<Thing> thingsToCreate)
+        {
+            var openedIteration = this.GetIteration();
+            if (openedIteration == null)
+            {
+                Console.WriteLine("At first an iteration should be opened");
```

Comment:
please return an exception, this should not happen and means that the method is used when it should not. I would use `InvalidOperationException`